go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/naag/terraform-provider-grafanacloud
```

Comment:
I think the module name is still referring to your personal repo?

Reply:
Fixed!